simba/station_optimization.py

```diff
@@ -171,7 +171,7 @@ def run_optimization(conf, sched=None, scen=None, args=None):
     # rotations itself.
     rotations_for_opt = optimizer.schedule.rotations.copy()
 
-    logger.log(msg="Starting greedy station optimization", level=100)
+    logger.log(msg="Starting greedy station optimization", level=39)
```

Comment:
maybe use a more verbose level? Like logging.WARN?

Reply:
my thinking was, to not use warn since it might be misleading. Its more like a "mandatory information"